main.go

```diff
@@ -193,6 +195,55 @@ func versionHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write(jsonVersion)
 }
 
+type heartbeat struct {
+	Status bool `json:"status"`
+	Checks struct {
+		CheckAutographHeartbeat bool `json:"check_autograph_heartbeat"`
+	} `json:"checks"`
+	Details string `json:"details"`
+}
+
+// send a GET request to the autograph heartbeat endpoint and
+// evaluate its status code before responding
+func heartbeatHandler(w http.ResponseWriter, r *http.Request) {
+	var st heartbeat
+	// assume the best, change if we encounter errors
+	st.Status = true
+	st.Checks.CheckAutographHeartbeat = true
+
+	u, err := url.Parse(conf.URL)
+	if err != nil {
+		log.Printf("failed to parse conf url %q: %v", conf.URL, err)
+		http.Error(w, "failed to parse conf URL", http.StatusInternalServerError)
```

Comment:
Do we plan to use Sentry for exception reporting in addition to this sort of response?